src/pages/messages/components/DoiRequestMessagesColumn.tsx

```diff
@@ -2,30 +2,38 @@ import { Typography } from '@mui/material';
 import { useTranslation } from 'react-i18next';
 import { ExpandedTicket, Ticket } from '../../../types/publication_types/ticket.types';
 import { toDateString } from '../../../utils/date-helpers';
+import { LastMessageBox } from './LastMessageBox';
 import { StyledMessagesContainer, StyledStatusMessageBox } from './PublishingRequestMessagesColumn';
 
 interface DoiRequestMessagesColumnProps {
   ticket: ExpandedTicket | Ticket;
+  showLastMessage?: boolean;
```

Comment:
liker ikke helt å skitne til med å introdusere sånne props for customization av hva som skal vises, da det har en tendens til å gjøre det vanskeligere å endre ting senere. Men ender ofte opp med å gjøre det likevel, fordi det virker litt overdrevent å splitte opp i to komponenter også, selv om det kanskje hadde vært det beste på lang sikt... 🤔

Reply:
Syns ikke det er noe problem her, men enig i at det ikke er bra i store mengder